src/man/authselect.8.adoc

```diff
@@ -52,13 +58,24 @@ To print help for the selected command run *authselect COMMAND --help*.
         The command will not print any informational message such as additional
         profile requirements or backup location. Errors are still being print.
 
-*apply-changes*::
+*apply-changes* [-b] [--backup=NAME]::
     Re-apply currently selected profile. If the profile templates were updated
     this command can be used to regenerate current system configuration in
     order to apply these changes on the system. This command will only re-apply
     the changes if the existing configuration is a valid authselect
     configuration, otherwise an error is returned.
 
+    *-b*:::
+        Backup system files before applying changes. The backup
+        will be stored at {AUTHSELECT_BACKUP_DIR}/NAME. Current time with
```

Comment:
Nitpicking: Should not be `stored as` instead of `stored at`?

Reply:
Given `NAME` is a directory, I think using `at` is correct. But I am not a native speaker.